### PR TITLE
LPS-186222 Update draft when portlet configuration changes

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:forward

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
Precioso

**Comment by ealonso** [comment]
ci:forward

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/reducers/draftReducer.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import * as TYPES from '../actions/types';
+
+export default function languageReducer(draft, action) {
```
Comment (david-truong) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/reducers/draftReducer.js:17]: should this be named draftReducer?

Comment (victorg1991) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/reducers/draftReducer.js:17]: Yes :) Thanks for the heads up, I'll update it

Comment (victorg1991) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/reducers/draftReducer.js:17]: It won't cause any error though

**Pull request closed without merging**